neurons/miners/StableMiner/base.py

```diff
@@ -397,14 +430,14 @@ def _base_priority(self, synapse) -> float:
         return priority
 
     def _base_blacklist(
-        self, synapse, vpermit_tao_limit=VPERMIT_TAO, rate_limit=1
+        self, synapse, vpermit_tao_limit=0.001, rate_limit=1
```

Comment:
Is this just for testing? If so we just need to remember to re-instate VPERMIT_TAO